modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -5549,6 +5549,49 @@ public void testGetScopeScopeKeyObjectEntriesPage() throws Exception {
 			_siteScopedObjectEntry1.getObjectEntryId());
 	}
 
+	@Test
+	public void testPatchCustomObjectEntryWithRequiredObjectField()
+		throws Exception {
+
+		ObjectField objectField1 = _objectFieldLocalService.getObjectField(
+			_objectDefinition1.getObjectDefinitionId(),
+			_OBJECT_FIELD_NAME_TEXT);
+		ObjectField objectField2 = _objectFieldLocalService.getObjectField(
+			_objectDefinition1.getObjectDefinitionId(), _OBJECT_FIELD_NAME_1);
```

Comment:
@devpedrotavares can you change to use the `_OBJECT_FIELD_NAME_2` here

Reply:
Hi @carolmariaabb , I just couldn't do the changes related to using `_OBJECT_FIELD_NAME_2`, because it actually belongs to another object definition